### PR TITLE
COMMERCE-4809 commerce-product-definitions-web fixed vocabularies vis…

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
…ibility for products

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
hey @andrea-ale-sbarra i don’t think this is right. Product assets are scoped to global site

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
@marco-leo I made a change after your review, can you have a look at that when you have a chance? thanks!

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/definition/details.jsp [hunk 1]
```diff
@@ -154,7 +154,8 @@ if ((cpDefinition != null) && (cpDefinition.getExpirationDate() != null)) {
 					<liferay-asset:asset-categories-selector
 						className="<%= CPDefinition.class.getName() %>"
 						classPK="<%= cpDefinitionId %>"
-						groupIds="<%= new long[] {company.getGroupId()} %>"
+						groupIds="<%= new long[] {company.getGroupId(), cpDefinitionsDisplayContext.getSiteGroupId()} %>"
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/definition/details.jsp:157]: @andrea-ale-sbarra I think the categories that we have to show are just the one connectected to global scope, not the one to the current site.  Why this was added?

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
as per the last comment from @marco-leo I close this pr, because the issue was resolved as not a bug

**Pull request closed without merging**